contracts/TokenHub.sol

```diff
@@ -96,6 +96,8 @@ contract TokenHub is ITokenHub, System, IParamSubscriber, IApplication, ISystemR
   uint256 constant public INIT_LOCK_PERIOD = 12 hours;
   // the lock period for large cross-chain transfer
   uint256 public lockPeriod;
+  // the lock pried for airdrop
+  uint256 constant public lockPeriodForAirdrop = 7 days;
```

Comment:
It is a const, so please use upper letter and put it with other const.

Reply:
fixed